src/js/components/Video/Video.js

```diff
@@ -80,6 +80,26 @@ const Video = forwardRef(
     const containerRef = useRef();
     const scrubberRef = useRef();
     const videoRef = useForwardedRef(ref);
+    const [newControls, setNewControls] = useState({
+      position: 'over',
```

Comment:
I think we should leverage the passed in controls prop more here:
`useState(typeof controls === 'string' ? { position: controls, items: [...] } : controls)`